### PR TITLE
Remove TLS and auth, code clean

### LINKED ISSUE #6: remove TLS and auth
TLS and auth should be removed from mailexporter and TLS and auth should be done by a tool specifically built for that (like authguard or an {nginx,apache,younameit}-reverseproxy) like in the rest of the prometheus-ecosystem.

### PR DESCRIPTION
**Pull request by dvrkps**
Closes #6

### CODE REVIEW

File: mailexporter.go [hunk 1]
```diff
@@ -294,10 +265,10 @@ func deleteMail(m email) {
 	promlog.Debug("rm ", m.filename)
 }
 
-// lateMail logs mails that have been so late that they timed out
-func lateMail(m email) {
```
Comment (cherti) [inline mailexporter.go:298]: Good idea renaming this function, but I'd prefer `handleLateMail`, as this function also performs filesystem-operations and not just logging. Can you change that name?

**Review by cherti — CHANGES_REQUESTED** [review]
Thank you very much! Looks good to me except for one little detail. :)
After adressing that we can get this merged.

File: mailexporter.go (hunk 1 above)
Comment (dvrkps) [reply mailexporter.go:298]: Sure.

**Review by cherti — APPROVED** [review]

**Comment by cherti** [comment]
Thank you very much, I'll merge once I had the time to test it.

**Pull request merged**